migrations/20200604110145_create_commit_table.ts

```diff
@@ -0,0 +1,20 @@
+import * as Knex from "knex";
+
+const commitTableName = "commit";
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.createTable(
+    commitTableName,
+    (builder: Knex.TableBuilder) => {
+      builder.increments("id").primary();
+      builder.text("message").notNullable();
+      builder.timestamp("committed_at");
+      builder.integer("user_id").unsigned().notNullable();
```

Comment:
Postgres does not have [unsigned](https://stackoverflow.com/questions/20810134/why-unsigned-integer-is-not-available-in-postgresql) integer. We can omit that.